docs/faq.md

```diff
@@ -1,45 +1,50 @@
 # OpenControl: Questions
 
-
 <script async class="speakerdeck-embed" data-id="1b3d6494fb12408dbe8f1cfe65eca769" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
 
-
 ## What?
 
 What license is this under?
 
 > The code portions are all licensed under Apache 2.0, except what has been contributed
-directly by the US Government, which is in the public domain. All written content
-has been licensed as Creative Commons Zero.
+directly by the US Government, which is in the public domain within the US. Internationally, the US Government licenses its code under [Creative Commons Zero 1.0](https://github.com/opencontrol/compliance-masonry/blob/master/LICENSE.md). All written content
+has been licensed as [Creative Commons Zero](https://creativecommons.org/publicdomain/zero/1.0/).
 
 What regulations can I use this for?
 
 > The principles behind OpenControl can be applied to any regulatory environment,
 or in fact any operational environment with running systems and software. Currently,
-the community repositories include FedRAMP low and moderate definitions using the
-NIST 800-53 certification, as well as a newly defined "LATO" authorization.
+the community repositories include FedRAMP Low and Moderate definitions using the
+NIST 800-53 certification, as well as a newly defined [Lightweight Authority to Operate](https://gsablogs.gsa.gov/innovation/2014/12/10/it-security-security-in-an-agile-development-cloud-world-by-kurt-garbars/) (LATO) NIST 800-53 baseline from the [General Services Administration](http://gsa.gov (GSA).
 
 What about PCI?
 
-What about SOX?
+> [We've got it!](https://github.com/opencontrol/compliance-masonry/blob/6c9a3ff81710dedb816ccb7d9edd1a50e84ce015/fixtures/standards/PCI-DSS-MAY-2015.yaml)
+
+What about FedRAMP High?
+
+> Coming soon.
 
 What about HIPAA?
 
+> Coming soon.
+
 What about ICD-503?
 
-What about FedRAMP?
+> No current plans - join the project and contribute!
```

Comment:
We're actually working on an early draft ICD-503, but probably won't be able to post it in the open. :/